airbyte-webapp/Dockerfile

```diff
@@ -4,5 +4,4 @@ FROM ${NGINX_IMAGE} as webapp
 EXPOSE 80
 
 COPY bin/build /usr/share/nginx/html
-COPY bin/docs /usr/share/nginx/html/docs
```

Comment:
:information_source: This is no longer needed, since the `copyDocs` and `copyDocAssets` tasks now copy their files directly into the `build/docker/bin/build/docs` folder instead of `build/docker/bin/docs` as beforehand. Thus `bin/build` contains the static content now as it's required to be served by nginx.